tests/ui/use_last.rs

```diff
@@ -0,0 +1,36 @@
+#![warn(clippy::use_last)]
+
+fn dont_use_last() -> Option<i32> {
+    let x = vec![2, 3, 5];
+    let last_element = x.get(x.len() - 1); // ~ERROR Use x.last()
+    last_element.map(|val| val + 1) // To avoid warnings
```

Comment:
you could just use `let _ = x.get ....`. The underscore silences the warnings, too